docs/advanced/operating-a-vsp.md

```diff
@@ -0,0 +1,42 @@
+# Operating a Voting Service Provider (VSP)
+
+---
+
+## Overview
+
+VSPs are a key part of the Decred infrastructure which make participating in proof-of-stake much more accessible for the average user. It is desirable to increase the number of operating VSPs in order to promote decentralization, improve the stability of the Decred network, and give stakeholders more choice.
+
+This is a high level guide of how to set up a VSP, and the steps required to list a VSP on [decred.org](https://decred.org/stakepools) and in Decrediton wallet. The guide is deliberately written at a high level and with minimal details because a level of system admin experience is assumed for VSP operators.
+
+---
+
+## Operator Requirements
+
+* Familiarity with system administration work on Linux, e.g. configuring and running nginx, MySQL, ssh, IPSec, duplicity, monit
+* Familiarity with compiling from source, setting up and maintaining `dcrd` and `dcrwallet`
+* Ability to effectively communicate in English
+* Availability to update VSP binaries when new releases are produced
+* Operators should be pairs of individuals or larger groups, e.g. corporate entities, so that the unavailability of a single person does not lead to extended outages in their absence.
+
+---
+
+## VSP Requirements
+
+There are several requirements for the VSP configuration:
+
+* At least one machine to host the web front end, handling web connections from VSP users and allowing them to configure their account.
+* At least three machines to host voting wallets. Necessary because at least one voting wallet needs to be connected to the network and ready to vote at any time.
```

Comment:
Please add a comment about each wallet using its own dcrd process as well.  I have seen 3 wallets using a single dcrd instance....

Reply:
Also some type of monitoring service is required as well.